doc/backend_vars.asciidoc

```diff
@@ -1,36 +1,71 @@
 Supported variables per backend
 -------------------------------
 
+.AMT backend
```

Comment:
why?

Reply:
https://progress.opensuse.org/issues/34303 is the reason

Reply:
updated PR